.github/workflows/kind.yml

```diff
@@ -153,7 +153,15 @@ jobs:
       - name: Run e2e tests
         run: |
           mkdir log
-          ANTREA_LOG_DIR=$PWD/log ./ci/kind/test-e2e-kind.sh
+          mkdir .coverage
+          ANTREA_LOG_DIR=$PWD/log ./ci/kind/test-e2e-kind.sh --coverage
+      - name: Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          file: .coverage/coverage-e2e.txt
+          flags: kind-e2e-tests
+          name: codecov-e2e-encap-test
```

Comment:
Unlike Antrea, Theia doesn't have `encap` mode.
So I recommend renaming this to `codecov-kind-e2e-test`